iree_tests/benchmarks/sdxl/conftest.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+def pytest_addoption(parser):
+    parser.addoption("--goldentime-rocm-e2e-ms", action="store", default=1661.5, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-unet-ms", action="store", default=450.5, type=float, help="Golden time to test benchmark")
```

Comment:
I think these defaults should be removed and the flag options should be required by all uses of the script. Otherwise, this encodes a set of values at this point in time for one GPU.